payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/autocomplete/PlacesClientProxy.kt

```diff
@@ -156,6 +166,7 @@ internal class UnsupportedPlacesClientProxy : PlacesClientProxy {
         if (BuildConfig.DEBUG) {
             throw exception
         }
+        errorReporter.report(ErrorReporter.UnexpectedErrorEvent.MISSING_PLACES_DEPENDENCY)
```

Comment:
This is less about a missing dependency, and more about us calling this method when the dependency is missing.

We support merchants using address element with and without the places dependency. 

Maybe a better name would be something like API call without places dependency (still not great, so feel free to use something different)